### ISSUE TITLE
pod/svclb-traefik does not correctly build on v1.22.2-rc1+k3s1 on arm device

### ISSUE THREAD

**Issue by rancher-max** [k3s-io/k3s#4048]
v1.22.2-rc1+k3s1 does not fully start it on my arm device: raspberry pi kernel `5.4.79-v7l+`

The containers `lb-port-80` and `lb-port-443` are both failing, both giving the same logs through kubectl and crictl:
```
standard_init_linux.go:228: exec user process caused: exec format error
```

Checking containerd.log doesn’t show anything useful either:
```
time="2021-09-20T09:10:56.106062697-07:00" level=info msg="CreateContainer within sandbox \"2cfa37c54e89238b57a7207f520c0e86fdf14697d4eef522db8cd9736463f62e\" for container &ContainerMetadata{Name:lb-port-80,Attempt:0,}"
time="2021-09-20T09:11:10.260892585-07:00" level=info msg="CreateContainer within sandbox \"2cfa37c54e89238b57a7207f520c0e86fdf14697d4eef522db8cd9736463f62e\" for &ContainerMetadata{Name:lb-port-80,Attempt:0,} returns container id \"967186d51a9e3930a8477c7d8a77b8b1bc0b75b72adbede55db29bec951c13e3\""
...
time="2021-09-20T09:11:10.597871508-07:00" level=info msg="CreateContainer within sandbox \"2cfa37c54e89238b57a7207f520c0e86fdf14697d4eef522db8cd9736463f62e\" for container &ContainerMetadata{Name:lb-port-443,Attempt:0,}"
time="2021-09-20T09:11:10.946204147-07:00" level=info msg="shim disconnected" id=967186d51a9e3930a8477c7d8a77b8b1bc0b75b72adbede55db29bec951c13e3
...
time="2021-09-20T09:11:15.296241638-07:00" level=info msg="CreateContainer within sandbox \"2cfa37c54e89238b57a7207f520c0e86fdf14697d4eef522db8cd9736463f62e\" for &ContainerMetadata{Name:lb-port-443,Attempt:0,} returns container id \"ff7c94a1931b60dc87007eef00db1848c027a7352b319c505073f59e0dfcc3ee\""
...
time="2021-09-20T09:11:17.184276875-07:00" level=info msg="CreateContainer within sandbox \"2cfa37c54e89238b57a7207f520c0e86fdf14697d4eef522db8cd9736463f62e\" for container &ContainerMetadata{Name:lb-port-80,Attempt:1,}"
time="2021-09-20T09:11:19.953052966-07:00" level=info msg="shim disconnected" id=ff7c94a1931b60dc87007eef00db1848c027a7352b319c505073f59e0dfcc3ee
```

Currently, we are building the arm stuff with the amd64 alpine image, and we need a different alpine image for the arm versions: `armv6`, `armv7`, `arm64 v8`

**Comment by rancher-max** [comment]
Validated on master branch using commit `28be0de4e81a20608f9dbb5729543a89aa936711`

Both fresh installs and upgrades work on my arm device.

**Comment by brandond** [comment]
Closing this since it was really just to track the regression for the revert PR. Can continue to track re-doing the fix in #4021
